app/services/authentication/providers/facebook.rb

```diff
@@ -9,7 +9,7 @@ def new_user_data
         {
           name: @info.name,
           email: @info.email || "",
-          remote_profile_image_url: @info.image,
+          remote_profile_image_url: @info.image.gsub("http", "https"),
```

Comment:
One question: If Facebook decides to revert back to `https` this would break again, right? Because it would turn `https://fb.com` into `httpss://fb.com`.

Maybe changing this to `@info.image.gsub("http://", "https://")` would still fix the problem and if they switch it back to `https` then it wouldn't find anything to replace and it would stay the same. Does this make sense?

Reply:
Ahh yes, great point! Will update.